projects/ngrx.io/content/guide/store/configuration/runtime-checks.md

```diff
@@ -50,10 +50,10 @@ To fix the above violation, a new reference to the state has to be created:
 
 ```ts
 export const reducer = createReducer(initialState,
-  on(addTodo, state => ({
+  on(addTodo, (state, { payload } => ({
```

Comment:
Same here

Reply:
These should be using `todo` instead of `payload`

Reply:
Done.